From f854ebf48e6362fa384689ac9b1ed5193ac456da Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Tue, 19 Aug 2014 15:27:38 -0400
Subject: [tools] perf/record: Move existing write_output into helper function

Message-id: <1408462094-14194-290-git-send-email-jolsa@redhat.com>
Patchwork-id: 88269
O-Subject: [PATCH RHEL7.1 BZ1131394 289/325] perf record: Move existing write_output into helper function
Bugzilla: 1131394
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1131394
https://bugzilla.redhat.com/show_bug.cgi?id=1131394

upstream
========
commit a9986fad6645b98d5bb3c2f83c22efb0761ca272
Author: David Ahern <dsahern@gmail.com>
Date: Thu Nov 7 21:23:24 2013 -0700

description
===========
Code move only; no logic changes. In preparation for the mmap based
output option in the next patch.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/builtin-record.c b/tools/perf/builtin-record.c
index 8f5af32..880227e 100644
--- a/tools/perf/builtin-record.c
+++ b/tools/perf/builtin-record.c
@@ -76,7 +76,7 @@ struct perf_record {
  long   samples;
 };
 
-static int write_output(struct perf_record *rec, void *buf, size_t size)
+static int do_write_output(struct perf_record *rec, void *buf, size_t size)
 {
  struct perf_data_file *file = &rec->file;
 
@@ -97,6 +97,11 @@ static int write_output(struct perf_record *rec, void *buf, size_t size)
  return 0;
 }
 
+static int write_output(struct perf_record *rec, void *buf, size_t size)
+{
+ return do_write_output(rec, buf, size);
+}
+
 static int process_synthesized_event(struct perf_tool *tool,
          union perf_event *event,
          struct perf_sample *sample __maybe_unused,
-- 
1.7.1